### ISSUE TITLE
Update projects readme files to link to MyGet Gallery

### ISSUE THREAD

**Issue by gep13** [cake-contrib/Home#5]
http://myget.org/gallery/cake-contrib

As suggested by @Jericho

**Comment by Jericho** [comment]
Here is an example of what I suggest:
```
| |Stable|Pre-release|
|:--:|:--:|:--:|
|GitHub Release|-|[![GitHub release](https://img.shields.io/github/release/cake-contrib/Cake.SendGrid.svg)](https://github.com/cake-contrib/Cake.SendGrid/releases/latest)|
|Package|[![NuGet](https://img.shields.io/nuget/v/Cake.SendGrid.svg)](https://www.nuget.org/packages/Cake.SendGrid)|[![MyGet](https://img.shields.io/myget/cake-contrib/vpre/Cake.SendGrid.svg)](http://myget.org/feed/cake-contrib/package/nuget/Cake.SendGrid)|
```
Which results in the following table:
![image](https://cloud.githubusercontent.com/assets/112710/25027566/9823cbbc-207b-11e7-90af-5e803727f1dc.png)

By the way, I'm not sure what the issue is with the "GitHub release" section of the table. It always shows the "none" badge for some reason.

**Comment by gep13** [comment]
@Jericho yeah, I think this entire section needs quite a bit of work.

The url:

https://github.com/cake-contrib/Cake.SendGrid/releases/latest

will only return something if you remove all the pre-release flags from here:

https://github.com/cake-contrib/Cake.SendGrid/releases

I guess the pre-release flag should only be set when the tag includes a `-beta` or `-rc` addition.

I think the shield should be moved from the `Pre-release` column to the `Stable` column, and perhaps simply have `n/a` in the pre-release column.

Thoughts?

**Comment by Jericho** [comment]
- Removing the pre-release flag on GitHub: I'm fine with that
- Moving the badge to the "Stable" column: makes sense
- "n/a" in the pre-release column: fine with me

It occurs to me: do we really need a table to present this information? What about 3 simple bullet points like this:
- Release Notes: \<badge to GitHub releases>
- Package: \<badge to Nuget package>
- Pre-release package: \<badge to myget package>

**Comment by Jericho** [comment]
If we want to keep the "table" format, what about something like this:

|||
|:--:|:--:|
|Release Notes|[![GitHub release](https://img.shields.io/github/release/cake-contrib/Cake.SendGrid.svg)](https://github.com/cake-contrib/Cake.SendGrid/releases/latest)|
|Package|[![NuGet](https://img.shields.io/nuget/v/Cake.SendGrid.svg)](https://www.nuget.org/packages/Cake.SendGrid)|
|Pre-release package|[![MyGet](https://img.shields.io/myget/cake-contrib/vpre/Cake.SendGrid.svg)](http://myget.org/feed/cake-contrib/package/nuget/Cake.SendGrid)|

**Comment by pascalberger** [comment]
IMHO there's a difference between pre-release packages from nuget.org and packages from myget. pre-release packages from nuget.org are official pre-relases (beta) targeted at users of the addin for testing purposes before a final release. Packages from myget I consider unstable development version, which can be useful for testing specific features or bugs, but are not supposed to be used by a wider audience.

**Comment by gep13** [comment]
@pascalberger yes, I don't disagree with this.  Perhaps since we can't easily get that information from a single URL on GitHub, perhaps `Pre-release package` becomes `CI Build`?
